autograd/numpy/fft.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import
+import numpy as np
```

Comment:
It's true that it doesn't matter for creating constants, but importing both np and anp could lead to bugs in future commits, since it's easy to accidentally use np. As @j-towns suggests, in general (when not just creating constants) it's important to use autograd.numpy to define vjp's so that autograd can be closed under its own operation. Also, there's no downside to using it; assigning into arrays isn't supported because it's not really compatible with reverse-mode autodiff to destroy information.